Updated Robot List

simonGoldstein opened the pull request:
### Description

The list of robots was missing many new robots that now use MoveIt. This Adds a bunch of new robots like Han's Robots elfin series, Aubo's i series, The Niryo One, and many more. In addition to adding photoes this adds links to their libraries where possible.

This also organizes the images/robots folder with company based sub-folders.

![NewRobots](https://user-images.githubusercontent.com/42970868/60622391-7b906e00-9d9d-11e9-9c6e-4fd956c04b23.png)

### Checklist
- [ ] Tested modified webpage locally using the ``build_locally.sh`` script
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

simonGoldstein commented:
I rebased to gh-pages and updated images in folders to no longer have the company name.

gavanderhoorn commented:
What is the rationale for including robots in the list for which no links to packages are then given?

For many industrial robots we could essentially list the entire product range, as most drivers are model-agnostic. As long as the controller is supported, MoveIt can be used with the robot -- provided there is a configuration package available. Isn't that what the purpose of the page is? To show robots that have such packages available?

rhaschke commented:
Looking at the commit history, I also noticed that you didn't rebase as announced, but you actually merged your changes. Please cleanup this PR. Finally, there should be a single commit only on top of gh-pages.

![image](https://user-images.githubusercontent.com/5376030/60884587-88a1d700-a24d-11e9-99c8-216ec500c16d.png)

simonGoldstein commented:
@gavanderhoorn I always figured the purpose of the page was to show off how many people used MoveIt, to instill confidence in unsure potential users that MoveIt was both widely used and safe. It may be that there are enough robots with repos that the other robots without aren't needed and can now be removed. I think that either I should remove all the no-repo robots or I should add the new ones, but I don't see much point in something in between. 

@rhaschke Yeah I appear to have royally bungled that up. Thanks for pointing that out. 

I was busy today but I should have some time either tomorrow or Friday to clean up this PR.

simonGoldstein commented:
I cleaned up the number of commits and I double checked that everything looks fine when I run ./build_locally. I think it is based on upstream/gh-pages and there shouldn't be any merge conflicts, but I am not 100% sure. Currently I just added more robots and have left the new and old robots on the list without repos alone.

rhaschke commented:
@simonGoldstein, this looks better. I added another commit, removing more images from `robots_old` folder (as you did before). However, there are still some images remaining in this deprecated folder. If you could clean them up as well (and fix the corresponding links spread throughout the website), that would be great.

gavanderhoorn commented:
@simonGoldstein wrote:
> I always figured the purpose of the page was to show off how many people used MoveIt, to instill confidence in unsure potential users that MoveIt was both widely used and safe.

Exaggerating a bit, but anyone could claim that Moveit works with a robot. At least according to this page no "proof" needs to be provided.

Additionally, it also looks like there is something special about moveit configurations for specific variants of -- say Fanucs. In reality, the MoveIt configuration pkgs are almost identical. Not listing, or actually listing a variant gives almost no additional information, apart from the fact that there is a configuration pkg available.

Seeing a robot on that page without then pointing to where the package can be found seems like a strange thing to me in that light.

simonGoldstein commented:
I emptied robots_old. The only reference left is in about/concepts and that was updated in a parallel PR.
